dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -653,6 +657,49 @@ public Limits setUserLimits(@ApiParam(hidden = true) @Auth User authUser,
         return limits;
     }
 
+    @POST
+    @Path("/{userId}/services/refresh")
+    @Timed
+    @UnitOfWork
+    @ApiOperation(value = "Syncs service data with Git accounts.", notes = "Currently only works with GitHub", authorizations = {
+            @Authorization(value = JWT_SECURITY_DEFINITION_NAME) },
+            response = Workflow.class, responseContainer = "List")
+    public List<Workflow> syncUserServices(@ApiParam(hidden = true) @Auth User authUser,
+            @ApiParam(value = "User ID", required = true) @PathParam("userId") Long userId) {
+        final User user = userDAO.findById(authUser.getId());
+        return syncAndGetServices(user, Optional.empty());
+    }
+
+    @POST
+    @Path("/{userId}/services/{organization}/refresh")
+    @Timed
+    @UnitOfWork
+    @ApiOperation(value = "Syncs services with Git accounts for a specified organization.",
+            authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) },
+            response = Workflow.class, responseContainer = "List")
+    public List<Workflow> syncUserServicesbyOrganization(@ApiParam(hidden = true) @Auth User authUser,
+            @ApiParam(value = "User ID", required = true) @PathParam("userId") Long userId,
+            @ApiParam(value = "Organization", required = true) @PathParam("organization") String organization) {
```

Comment:
Is this the name or organization id?  Could use an example in swagger.

Reply:
It's the name. Made it clearer in new commit.